templates/sectionsummary.html

```diff
@@ -26,8 +26,10 @@ <h1>{{content.summary.title}}</h1>
     <h1>{{content.summary.title}}</h1>
     {% for summary in content.summary.custom_summary %}
       {% if summary.type == 'List' %}
-
-          <div {{'class="u-mb-l"' if not loop.last }} data-qa="{{summary.list_name}}-list-summary">
+          {% set add_link = summary.add_link %}
+          {% set add_link_text = summary.add_link_text %}
+          {% set list_name = summary.list_name %}
+          <div {{'class="u-mb-l"' if not loop.last }} data-qa="{{list_name}}-list-summary">
           <h2>{{summary.title}}</h2>
           {% if summary.list and summary.list.list_items %}
```

Comment:
There is already a check for an empty list in `list-summary.html` that checks the length of `list_items`. We should consider moving the empty list logic out of this template and into that one.

Reply:
This has been overhauled using Mebin's code below, so this doesn't use length of `list_items` anymore.